.github/workflows/issue_opened.yml

```diff
@@ -0,0 +1,33 @@
+name: Issue Opened
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    permissions: {}
+    if: ${{ !contains(fromJSON('["MEMBER", "OWNER"]'), github.event.issue.author_association) }}
+    steps:
+      - name: Run webhook curl command
+        env:
+          WEBHOOK_URL: ${{ secrets.SLACK_ISSUE_WEBHOOK_URL }}
+          ISSUE: ${{toJson(github.event.issue.title)}}
+          ISSUE_URL: ${{github.event.issue.html_url}}
+          USER: ${{github.event.issue.user.login}}
+        shell: bash
+        run: echo $ISSUE | sed 's/[^a-zA-Z0-9 &().,:]//g' | xargs -I {} curl -s POST "$WEBHOOK_URL" -H "Content-Type:application/json" --data '{"issue":"{}", "issueUrl":"'$ISSUE_URL'", "user":"'$USER'"}'
+
+  maintainer-opened:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    if: ${{ contains(fromJSON('["MEMBER", "OWNER"]'), github.event.issue.author_association) }}
+    steps:
+      - name: Post comment if maintainer opened.
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ISSUE_NUMBER: ${{ github.event.issue.number }}
+        run: |
+          gh issue comment $ISSUE_NUMBER --repo aws-amplify/amplify-swift -b "This issue was opened by a maintainer of this repository; updates will be posted here. If you are also experiencing this issue, please comment here with any relevant information so that we're aware and can prioritize accordingly."
```

Comment:
does this need to be updated? `--repo aws-amplify/amplify-swift`

Reply:
```suggestion
          gh issue comment $ISSUE_NUMBER --repo aws-amplify/amplify-ui-swift-authenticator -b "This issue was opened by a maintainer of this repository; updates will be posted here. If you are also experiencing this issue, please comment here with any relevant information so that we're aware and can prioritize accordingly."
```